### PR TITLE
Fix the build issue

### PR DESCRIPTION
**Pull request by mkumatag**
Fixes: https://github.com/openshift/installer/issues/1931

### CODE REVIEW

**Comment by abhinavdahiya** [comment]
Please make sure the commit follows the guidelines from https://github.com/openshift/installer/blob/master/CONTRIBUTING.md#commit-message-format

File: pkg/destroy/libvirt/libvirt.go [hunk 1]
```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/sirupsen/logrus"
 
 	"github.com/openshift/installer/pkg/destroy/providers"
+	"github.com/openshift/installer/pkg/types"
```
Comment (abhinavdahiya) [inline pkg/destroy/libvirt/libvirt.go]: This seems not required..

Comment (mkumatag) [reply pkg/destroy/libvirt/libvirt.go]: This does require because there it is been used in line number 50 in `New` function.

**Comment by mkumatag** [comment]
> Please make sure the commit follows the guidelines from https://github.com/openshift/installer/blob/master/CONTRIBUTING.md#commit-message-format

fixed that, please refer the latest amended commit.

**Comment by csrwng** [comment]
@mkumatag thanks for fixing this. I erroneously assumed that libvirt was included when doing a regular build so I thought it was working ok.

**Comment by mkumatag** [comment]
> @mkumatag thanks for fixing this. I erroneously assumed that libvirt was included when doing a regular build so I thought it was working ok.

I see libvirt files got `+build libvirt` tags mentioned, so will have to pass the `libvirt` tag while building, I insist on installing all the prereq libraries mention the `libvirt` tag while building to cover all this code as well.

**Comment by markmc** [comment]
Patch fixes the build for me

/lgtm

**Comment by zeenix** [comment]
> @mkumatag thanks for fixing this. I erroneously assumed that libvirt was included when doing a regular build so I thought it was working ok.

We need working `e2e-libvirt` enabled by default to avoid these breakages. I hope once we've fixed [the current issues](https://github.com/openshift/installer/issues/1893), we can enable it.

Talking of which, I'll launch that check for just testing the build part cause currently it will fail because of issue#1893.

/test e2e-libvirt

**Comment by mkumatag** [comment]
can someone approve this PR? @abhinavdahiya @enxebre @praveenkumar @zeenix

**Pull request merged**